docs/migrating-from-chuck.md

```diff
@@ -0,0 +1,126 @@
+# Migrating from Chuck
+
+Please refer to this page if you're **migrating from [chuck](https://github.com/jgilfelt/chuck) to [chucker](https://github.com/ChuckerTeam/chucker)**
+
+The following guide is written assuming you're planning to use Chucker version 3.0.0.
+
+## 1. Update the Gradle dependency
+
+If you're using Chuck you need to update your gradle `dependencies` block from this:
+
+```groovy
+dependencies {
+    debugCompile 'com.readystatesoftware.chuck:library:1.1.0'
+    releaseCompile 'com.readystatesoftware.chuck:library-no-op:1.1.0'
+}
+```
+
+to this:
+
+```groovy
+repositories {
+    // Other repositories here.
+    maven { url 'https://jitpack.io' }
```

Comment:
Consistency of quotes: please replace `'` with `"` (also on the other occurrences)